src/Dataflow/Network/Module.cc

```diff
@@ -471,6 +483,20 @@ bool Module::executeWithSignals() NOEXCEPT
   return impl_->returnCode_;
 }
 
+void Module::runProgrammablePortInput()
+{
+  auto prog = getOptionalInputAtIndex<MetadataObject>(ProgrammablePortId());
+  if (prog && *prog)
+  {
+    //logCritical("MetadataObject found! {}", id().id_);
+    (*prog)->process(id());
+  }
+  else
+  {
+    //logCritical("\tMetadataObject NOT found! {}", id().id_);
```

Comment:
Temporary? Not sure if the else statement is needed here unless there is a future use for this.

Reply:
I'll need to change the log level of those print outs. Then the else block can stay.